src/html.sortable.src.js

```diff
@@ -14,30 +14,170 @@
  */
 var dragging;
 var draggingHeight;
-var placeholders = $();
+var placeholders = [];
 var sortables = [];
+/**
+ * Get or set data on element
+ * @param {Element} element
+ * @param {string} key
+ * @param {*} value
+ * @return {*}
+ */
+var _data = function(element, key, value) {
+  if (value === undefined) {
+    return element && element.h5s && element.h5s.data && element.h5s.data[key];
+  } else {
+    element.h5s = element.h5s || {};
+    element.h5s.data = element.h5s.data || {};
+    element.h5s.data[key] = value;
+  }
+};
+/**
+ * Remove data from element
+ * @param {Element} element
+ */
+var _removeData = function(element) {
+  if (element.h5s) {
+    delete element.h5s.data;
+  }
+};
+var matches;
+switch (true) {
+  case 'matches' in window.Element.prototype:
+    matches = 'matches';
+    break;
+  case 'msMatchesSelector' in window.Element.prototype:
+    matches = 'msMatchesSelector';
+    break;
+  case 'webkitMatchesSelector' in window.Element.prototype:
+    matches = 'webkitMatchesSelector';
+    break;
+}
+/**
+ * Filter only wanted nodes
+ * @param {Array|NodeList} nodes
+ * @param {Array/string} wanted
+ * @returns {Array}
+ * @private
+ */
+var _filter = function(nodes, wanted) {
+  if (!wanted) {
+    return Array.prototype.slice.call(nodes);
+  }
+  var result = [];
+  for (var i = 0; i < nodes.length; ++i) {
+    if (typeof wanted === 'string' && nodes[i][matches](wanted)) {
+      result.push(nodes[i]);
+    }
+    if (wanted.indexOf(nodes[i]) !== -1) {
+      result.push(nodes[i]);
+    }
+  }
+  return result;
+};
+/**
+ * @param {Array|Element} element
+ * @param {Array|string} event
+ * @param {Function} callback
+ */
+var _on = function(element, event, callback) {
+  var i;
+  var events;
+
+  if (element instanceof Array) {
+    for (i = 0; i < element.length; ++i) {
+      _on(element[i], event, callback);
+    }
+    return;
+  }
+  if (event.indexOf(' ') !== -1) {
+    for (i = 0, events = event.split(' '); i < events.length; ++i) {
+      _on(element, events[i], callback);
+    }
+    return;
+  }
+  element.addEventListener(event, callback);
+  element.h5s = element.h5s || {};
+  element.h5s.events = element.h5s.events || {};
+  element.h5s.events[event] = callback;
+};
+/**
+ * @param {Array|Element} element
+ * @param {Array|string} event
+ */
+var _off = function(element, event) {
+  var i;
+  var events;
+
+  if (element instanceof Array) {
+    for (i = 0; i < element.length; ++i) {
+      _off(element[i], event);
+    }
+    return;
+  }
+  if (event.indexOf(' ') !== -1) {
+    for (i = 0, events = event.split(' '); i < events.length; ++i) {
+      _off(element, events[i]);
+    }
+    return;
+  }
+  if (element.h5s && element.h5s.events && element.h5s.events[event]) {
```

Comment:
I think we can drop support for space-separated events list, but I'd like to keep first `if` so that it would be possible to remove event handler from array of events (which will become quite lengthy otherwise). What you think?